src/main/java/kr/codesquad/model/Ladder.java

```diff
@@ -9,36 +15,67 @@ public Ladder(Random random) {
         this.random = random;
     }
 
-    public String[][] makeLadder(String joinMember, String maxLadderHeight) {
-        String[][] ladder = new String[Integer.parseInt(maxLadderHeight)][
-                Integer.parseInt(joinMember) + Integer.parseInt(joinMember) - 1];
-        makeColumn(ladder);
+    public List<List<String>> make(String joinMembers, int maxLadderHeight) {
+        List<String> joinMembersSplit = new ArrayList<>(Arrays.asList(joinMembers.split(",")));
+        List<List<String>> ladder = new ArrayList<>();
+        init(ladder, maxLadderHeight, joinMembersSplit.size());
```

Comment:
```suggestion
        List<List<String>> ladder = init(maxLadderHeight, joinMembersSplit.size());
```

요것도 이렇게 줄일 수 있을 것 같아요.